lib/api_client.ex

```diff
@@ -12,13 +12,14 @@ defmodule GlassFactoryApi.ApiClient do
       {:ok , %{body: "[]", headers: [], status_code: 200}}
 
   If no configuration is passed, it will look for them in `Configuration.build\\1`
+  If no query string is passed, no filter is applied.
   """
 
-  @spec get(String.t(), map()) :: Tesla.Env.t()
-  def get(resource, configuration) do
+  @spec get(String.t(), Keyword.t(), map()) :: Tesla.Env.t()
+  def get(resource, query_string \\ [], configuration) do
```

Comment:
I'd call this `params` instead of `query_string`, because it is not a string yet at this point.

Reply:
Or maybe just `query`